src/main/resources/db/migration/V1_13__sentence_adjustment.sql

```diff
@@ -0,0 +1,11 @@
+create table sentence_adjustment_mapping
+(
+    nomis_sentence_adjustment_id   bigint       not null    UNIQUE,
+    sentence_adjustment_id         bigint       not null    PRIMARY KEY,
```

Comment:
BTW we don't what this key will look like yet it might be UUID - but obviously this is fine for now